### ISSUE TITLE
add cluster when there is a lot of pin in the same place

### ISSUE THREAD

**Issue by ViolaineDo** [stevage/VMap#2]
currently some pins are overlaid, would be great to have clusters

**Comment by stevage** [comment]
What is the goal here? Is there an example where overlapping pins is a problem that clusters can solve?

**Comment by ViolaineDo** [comment]
Yes here at hopital du Taaone we have 7 pins, would be better to have a cluster  instead.

![Capture du 2020-04-15 07-26-05](https://user-images.githubusercontent.com/12017078/79368000-70e93480-7eea-11ea-93d3-7ef7bb4a8650.png)
